molecule/default/requirements.yml

```diff
@@ -1,4 +1,6 @@
 ---
 - src: elastic.elasticsearch
-  version: 5.5.1
+  version: 6.4.0
```

Comment:
https://github.com/entercloudsuite/ansible-elasticsearch/blob/87f6da17be147bf6964fa86bd9034289084257ee/molecule/default/requirements.yml#L2-L3

line 2-3 are not needed, use the local repo instead. See above